src/utils/constants.ts

```diff
@@ -19,6 +19,7 @@ const NOTION_STATUS_OPEN = 'Open';
 const RETRO_FINISHED_MESSAGE =
   "We're done here! All attendees have provided their update. See you next week!";
 const RETRO_NEXT_SPEAKER_MESSAGE = "you're next! Please provide your update.";
+const SUPPORT_IDENTIFIER = '#question';
```

Comment:
need a better name for this variable. since this is a question for chatgpt3, maybe word the variable name accordingly